Content.Shared/Atmos/Components/GasPressurePumpComponent.cs

```diff
@@ -24,4 +24,10 @@ public sealed partial class GasPressurePumpComponent : Component
     [DataField]
     [GuidebookData]
     public float MaxTargetPressure = Atmospherics.MaxOutputPressure;
+
+    /// <summary>
+    /// Frontier - Start the pump with the map.
+    /// </summary>
+    [DataField]
+    public bool StartOnMapInit { get; set; }
```

Comment:
Consistency with `GasFilterComponent` and `GasMixerComponent.cs`

```suggestion
    public bool StartOnMapInit { get; set; } = false;
```